### ISSUE TITLE
lookup_users

### ISSUE THREAD

**Issue by nicoleprause** [ropensci/rtweet#626]
### Problem

lookup_users never successful, "Error: $ operator is invalid for atomic vectors"

### Expected behavior

I gather Twitter user ID's using get_followers. I use the new lookup_users expecting to receive information (e.g., screen_name, location, etc.) associated with each user_id.

### Reproduce the problem

```r
CNN<-get_followers("CNN", 
                            retryonratelimit = TRUE )
lookup_users(
    CNN$user_id,
       parse=TRUE,
       token=NULL,
       retryonratelimit = TRUE,
       verbose=TRUE
     )

Error: $ operator is invalid for atomic vectors 

```

### rtweet version

``` r
## copy/paste output
packageVersion("rtweet")
‘0.7.0.9000’
```

### Session info

``` r
## copy/paste output
sessionInfo()
attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] rtweet_0.7.0.9000  magrittr_2.0.1     stringr_1.4.0      scales_1.1.1       plyr_1.8.6         RColorBrewer_1.1-2 waffle_0.7.0      
 [8] writexl_1.4.0      ggplot2_3.3.5      readxl_1.3.1      

loaded via a namespace (and not attached):
 [1] progress_1.2.2    tinytex_0.33      tidyselect_1.1.1  xfun_0.24         remotes_2.4.0     purrr_0.3.4       colorspace_2.0-2  vctrs_0.3.8      
 [9] generics_0.1.0    utf8_1.2.2        rlang_0.4.11      pillar_1.6.2      later_1.2.0       glue_1.4.2        withr_2.4.2       DBI_1.1.1        
[17] rappdirs_0.3.3    bit64_4.0.5       lifecycle_1.0.0   munsell_0.5.0     gtable_0.3.0      cellranger_1.1.0  extrafont_0.17    httpuv_1.6.3     
[25] curl_4.3.2        fansi_0.5.0       Rttf2pt1_1.3.9    Rcpp_1.0.7        openssl_1.4.4     promises_1.2.0.1  jsonlite_1.7.2    bit_4.0.4        
[33] gridExtra_2.3     askpass_1.1       hms_1.1.0         stringi_1.7.3     dplyr_1.0.7       grid_4.1.0        cli_3.0.1         tools_4.1.0      
[41] tibble_3.1.3      crayon_1.4.1      extrafontdb_1.0   pkgconfig_2.0.3   ellipsis_0.3.2    prettyunits_1.1.1 assertthat_0.2.1  httr_1.4.2       
[49] rstudioapi_0.13   R6_2.5.0          compiler_4.1.0   
```

**Comment by llrs** [comment]
Hi thanks for the report.

I see you used the devel version of rtweet but when I tried this code (only changing to `retryonratelimit = FALSE` with the current code and I couldn't reproduce this error (but found a different error :upside_down_face: ). Could you please show which commit are you using? Use `devtools::session_info()` and it will appear.  
Check that you copy all the session information (on the current output I don't see which version of R are you using or the OS).

**Comment by nicoleprause** [comment]
I just updated devtools and get a more exciting (?) error "Error in FUN(X[[i]], ...) : withheld_in_countries " with the same code run above. 

My devtools::session_info() is:

─ Session info 
 setting  value                       
 version  R version 4.1.0 (2021-05-18)
 os       macOS Big Sur 10.16         
 system   x86_64, darwin17.0          
 ui       RStudio                     
 language (EN)                        
 collate  en_US.UTF-8                 
 ctype    en_US.UTF-8                 
 tz       America/[redacted]        
 date     2021-09-18                  

─ Packages 
 package      * version    date       lib source                          
 askpass        1.1        2019-01-13 [1] CRAN (R 4.1.0)                  
 assertthat     0.2.1      2019-03-21 [1] CRAN (R 4.1.0)                  
 bit            4.0.4      2020-08-04 [1] CRAN (R 4.1.0)                  
 bit64          4.0.5      2020-08-30 [1] CRAN (R 4.1.0)                  
 cachem         1.0.6      2021-08-19 [1] CRAN (R 4.1.0)                  
 callr          3.7.0      2021-04-20 [1] CRAN (R 4.1.0)                  
 cellranger     1.1.0      2016-07-27 [1] CRAN (R 4.1.0)                  
 cli            3.0.1      2021-07-17 [1] CRAN (R 4.1.0)                  
 colorspace     2.0-2      2021-06-24 [1] CRAN (R 4.1.0)                  
 crayon         1.4.1      2021-02-08 [1] CRAN (R 4.1.0)                  
 curl           4.3.2      2021-06-23 [1] CRAN (R 4.1.0)                  
 DBI            1.1.1      2021-01-15 [1] CRAN (R 4.1.0)                  
 desc           1.3.0      2021-03-05 [1] CRAN (R 4.1.0)                  
 devtools     * 2.4.2      2021-06-07 [1] CRAN (R 4.1.0)                  
 dplyr          1.0.7      2021-06-18 [1] CRAN (R 4.1.0)                  
 ellipsis       0.3.2      2021-04-29 [1] CRAN (R 4.1.0)                  
 extrafont      0.17       2014-12-08 [1] CRAN (R 4.1.0)                  
 extrafontdb    1.0        2012-06-11 [1] CRAN (R 4.1.0)                  
 fansi          0.5.0      2021-05-25 [1] CRAN (R 4.1.0)                  
 fastmap        1.1.0      2021-01-25 [1] CRAN (R 4.1.0)                  
 fs             1.5.0      2020-07-31 [1] CRAN (R 4.1.0)                  
 generics       0.1.0      2020-10-31 [1] CRAN (R 4.1.0)                  
 ggplot2      * 3.3.5      2021-06-25 [1] CRAN (R 4.1.0)                  
 glue           1.4.2      2020-08-27 [1] CRAN (R 4.1.0)                  
 gridExtra      2.3        2017-09-09 [1] CRAN (R 4.1.0)                  
 gtable         0.3.0      2019-03-25 [1] CRAN (R 4.1.0)                  
 hms            1.1.0      2021-05-17 [1] CRAN (R 4.1.0)                  
 httpuv         1.6.3      2021-09-09 [1] CRAN (R 4.1.0)                  
 httr           1.4.2      2020-07-20 [1] CRAN (R 4.1.0)                  
 jsonlite       1.7.2      2020-12-09 [1] CRAN (R 4.1.0)                  
 later          1.2.0      2021-04-23 [1] CRAN (R 4.1.0)                  
 lifecycle      1.0.0      2021-02-15 [1] CRAN (R 4.1.0)                  
 magrittr     * 2.0.1      2020-11-17 [1] CRAN (R 4.1.0)                  
 memoise        2.0.0      2021-01-26 [1] CRAN (R 4.1.0)                  
 munsell        0.5.0      2018-06-12 [1] CRAN (R 4.1.0)                  
 openssl        1.4.4      2021-04-30 [1] CRAN (R 4.1.0)                  
 pillar         1.6.2      2021-07-29 [1] CRAN (R 4.1.0)                  
 pkgbuild       1.2.0      2020-12-15 [1] CRAN (R 4.1.0)                  
 pkgconfig      2.0.3      2019-09-22 [1] CRAN (R 4.1.0)                  
 pkgload        1.2.2      2021-09-11 [1] CRAN (R 4.1.0)                  
 plyr         * 1.8.6      2020-03-03 [1] CRAN (R 4.1.0)                  
 prettyunits    1.1.1      2020-01-24 [1] CRAN (R 4.1.0)                  
 processx       3.5.2      2021-04-30 [1] CRAN (R 4.1.0)                  
 progress       1.2.2      2019-05-16 [1] CRAN (R 4.1.0)                  
 promises       1.2.0.1    2021-02-11 [1] CRAN (R 4.1.0)                  
 ps             1.6.0      2021-02-28 [1] CRAN (R 4.1.0)                  
 purrr          0.3.4      2020-04-17 [1] CRAN (R 4.1.0)                  
 R6             2.5.0      2020-10-28 [1] CRAN (R 4.1.0)                  
 rappdirs       0.3.3      2021-01-31 [1] CRAN (R 4.1.0)                  
 RColorBrewer * 1.1-2      2014-12-07 [1] CRAN (R 4.1.0)                  
 Rcpp           1.0.7      2021-07-07 [1] CRAN (R 4.1.0)                  
 readxl       * 1.3.1      2019-03-13 [1] CRAN (R 4.1.0)                  
 remotes        2.4.0      2021-06-02 [1] CRAN (R 4.1.0)                  
 rlang          0.4.11     2021-04-30 [1] CRAN (R 4.1.0)                  
 rprojroot      2.0.2      2020-11-15 [1] CRAN (R 4.1.0)                  
 rstudioapi     0.13       2020-11-12 [1] CRAN (R 4.1.0)                  
 Rttf2pt1       1.3.9      2021-07-22 [1] CRAN (R 4.1.0)                  
 rtweet       * 0.7.0.9000 2021-09-17 [1] Github (ropensci/rtweet@69ae100)
 scales       * 1.1.1      2020-05-11 [1] CRAN (R 4.1.0)                  
 sessioninfo    1.1.1      2018-11-05 [1] CRAN (R 4.1.0)                  
 stringi        1.7.3      2021-07-16 [1] CRAN (R 4.1.0)                  
 stringr      * 1.4.0      2019-02-10 [1] CRAN (R 4.1.0)                  
 testthat       3.0.4      2021-07-01 [1] CRAN (R 4.1.0)                  
 tibble         3.1.3      2021-07-23 [1] CRAN (R 4.1.0)                  
 tidyselect     1.1.1      2021-04-30 [1] CRAN (R 4.1.0)                  
 tinytex        0.33       2021-08-05 [1] CRAN (R 4.1.0)                  
 usethis      * 2.0.1      2021-02-10 [1] CRAN (R 4.1.0)                  
 utf8           1.2.2      2021-07-24 [1] CRAN (R 4.1.0)                  
 vctrs          0.3.8      2021-04-29 [1] CRAN (R 4.1.0)                  
 waffle       * 0.7.0      2017-01-07 [1] CRAN (R 4.1.0)                  
 withr          2.4.2      2021-04-18 [1] CRAN (R 4.1.0)                  
 writexl      * 1.4.0      2021-04-20 [1] CRAN (R 4.1.0)                  
 xfun           0.24       2021-06-15 [1] CRAN (R 4.1.0)

**Comment by nicoleprause** [comment]
I noticed my R version was from May, so I updated to the most recent released in August. Same error "Error in FUN(X[[i]], ...) : withheld_in_countries" with the sample code. 

─ Session info  setting  value                       
 version  R version 4.1.1 (2021-08-10)
 os       macOS Big Sur 10.16         
 system   x86_64, darwin17.0          
 ui       RStudio                     
 language (EN)                        
 collate  en_US.UTF-8                 
 ctype    en_US.UTF-8                 
 tz       America/[redacted]  
 date     2021-09-18                  

─ Packages 
package     * version    date       lib source                          
 askpass       1.1        2019-01-13 [1] CRAN (R 4.1.0)                  
 assertthat    0.2.1      2019-03-21 [1] CRAN (R 4.1.0)                  
 cachem        1.0.6      2021-08-19 [1] CRAN (R 4.1.0)                  
 callr         3.7.0      2021-04-20 [1] CRAN (R 4.1.0)                  
 cli           3.0.1      2021-07-17 [1] CRAN (R 4.1.0)                  
 colorspace    2.0-2      2021-06-24 [1] CRAN (R 4.1.0)                  
 crayon        1.4.1      2021-02-08 [1] CRAN (R 4.1.0)                  
 curl          4.3.2      2021-06-23 [1] CRAN (R 4.1.0)                  
 DBI           1.1.1      2021-01-15 [1] CRAN (R 4.1.0)                  
 desc          1.3.0      2021-03-05 [1] CRAN (R 4.1.0)                  
 devtools      2.4.2      2021-06-07 [1] CRAN (R 4.1.0)                  
 dplyr         1.0.7      2021-06-18 [1] CRAN (R 4.1.0)                  
 ellipsis      0.3.2      2021-04-29 [1] CRAN (R 4.1.0)                  
 fansi         0.5.0      2021-05-25 [1] CRAN (R 4.1.0)                  
 fastmap       1.1.0      2021-01-25 [1] CRAN (R 4.1.0)                  
 fs            1.5.0      2020-07-31 [1] CRAN (R 4.1.0)                  
 generics      0.1.0      2020-10-31 [1] CRAN (R 4.1.0)                  
 ggplot2       3.3.5      2021-06-25 [1] CRAN (R 4.1.0)                  
 glue          1.4.2      2020-08-27 [1] CRAN (R 4.1.0)                  
 gtable        0.3.0      2019-03-25 [1] CRAN (R 4.1.0)                  
 hms           1.1.0      2021-05-17 [1] CRAN (R 4.1.0)                  
 httr          1.4.2      2020-07-20 [1] CRAN (R 4.1.0)                  
 jsonlite      1.7.2      2020-12-09 [1] CRAN (R 4.1.0)                  
 lifecycle     1.0.0      2021-02-15 [1] CRAN (R 4.1.0)                  
 magrittr      2.0.1      2020-11-17 [1] CRAN (R 4.1.0)                  
 memoise       2.0.0      2021-01-26 [1] CRAN (R 4.1.0)                  
 munsell       0.5.0      2018-06-12 [1] CRAN (R 4.1.0)                  
 openssl       1.4.4      2021-04-30 [1] CRAN (R 4.1.0)                  
 pillar        1.6.2      2021-07-29 [1] CRAN (R 4.1.0)                  
 pkgbuild      1.2.0      2020-12-15 [1] CRAN (R 4.1.0)                  
 pkgconfig     2.0.3      2019-09-22 [1] CRAN (R 4.1.0)                  
 pkgload       1.2.2      2021-09-11 [1] CRAN (R 4.1.0)                  
 prettyunits   1.1.1      2020-01-24 [1] CRAN (R 4.1.0)                  
 processx      3.5.2      2021-04-30 [1] CRAN (R 4.1.0)                  
 progress      1.2.2      2019-05-16 [1] CRAN (R 4.1.0)                  
 ps            1.6.0      2021-02-28 [1] CRAN (R 4.1.0)                  
 purrr         0.3.4      2020-04-17 [1] CRAN (R 4.1.0)                  
 R6            2.5.0      2020-10-28 [1] CRAN (R 4.1.0)                  
 remotes       2.4.0      2021-06-02 [1] CRAN (R 4.1.0)                  
 rlang         0.4.11     2021-04-30 [1] CRAN (R 4.1.0)                  
 rprojroot     2.0.2      2020-11-15 [1] CRAN (R 4.1.0)                  
 rstudioapi    0.13       2020-11-12 [1] CRAN (R 4.1.0)                  
 rtweet      * 0.7.0.9000 2021-09-17 [1] Github (ropensci/rtweet@69ae100)
 scales        1.1.1      2020-05-11 [1] CRAN (R 4.1.0)                  
 sessioninfo   1.1.1      2018-11-05 [1] CRAN (R 4.1.0)                  
 testthat      3.0.4      2021-07-01 [1] CRAN (R 4.1.0)                  
 tibble        3.1.3      2021-07-23 [1] CRAN (R 4.1.0)                  
 tidyselect    1.1.1      2021-04-30 [1] CRAN (R 4.1.0)                  
 tinytex       0.33       2021-08-05 [1] CRAN (R 4.1.0)                  
 usethis       2.0.1      2021-02-10 [1] CRAN (R 4.1.0)                  
 utf8          1.2.2      2021-07-24 [1] CRAN (R 4.1.0)                  
 vctrs         0.3.8      2021-04-29 [1] CRAN (R 4.1.0)                  
 withr         2.4.2      2021-04-18 [1] CRAN (R 4.1.0)                  
 xfun          0.24       2021-06-15 [1] CRAN (R 4.1.0)

**Comment by nicoleprause** [comment]
Replicated behavior in another environment

- Session info -----------------------------------------------------------------
 setting  value                       
 version  R version 4.1.0 (2021-05-18)
 os       Windows 10 x64              
 system   x86_64, mingw32             
 ui       RStudio                     
 language (EN)                        
 collate  English_United States.1252  
 ctype    English_United States.1252  
 tz       America/[redacted]       
 date     2021-09-21                  

- Packages ---------------------------------------------------------------------
 package      * version    date       lib source                          
 askpass        1.1        2019-01-13 [1] CRAN (R 4.1.0)                  
 assertthat     0.2.1      2019-03-21 [1] CRAN (R 4.1.0)                  
 backports      1.2.1      2020-12-09 [1] CRAN (R 4.1.0)                  
 base64enc      0.1-3      2015-07-28 [1] CRAN (R 4.1.0)                  
 bitops         1.0-7      2021-04-24 [1] CRAN (R 4.1.0)                  
 cachem         1.0.5      2021-05-15 [1] CRAN (R 4.1.0)                  
 callr          3.7.0      2021-04-20 [1] CRAN (R 4.1.0)                  
 checkmate      2.0.0      2020-02-06 [1] CRAN (R 4.1.0)                  
 cli            3.0.1      2021-07-17 [1] CRAN (R 4.1.0)                  
 cluster        2.1.2      2021-04-17 [2] CRAN (R 4.1.0)                  
 colorspace     2.0-2      2021-06-24 [1] CRAN (R 4.1.0)                  
 crayon         1.4.1      2021-02-08 [1] CRAN (R 4.1.0)                  
 curl           4.3.2      2021-06-23 [1] CRAN (R 4.1.0)                  
 data.table     1.14.0     2021-02-21 [1] CRAN (R 4.1.0)                  
 DBI            1.1.1      2021-01-15 [1] CRAN (R 4.1.0)                  
 desc           1.3.0      2021-03-05 [1] CRAN (R 4.1.0)                  
 devtools     * 2.4.2      2021-06-07 [1] CRAN (R 4.1.1)                  
 digest         0.6.27     2020-10-24 [1] CRAN (R 4.1.0)                  
 dplyr        * 1.0.7      2021-06-18 [1] CRAN (R 4.1.0)                  
 ellipsis       0.3.2      2021-04-29 [1] CRAN (R 4.1.0)                  
 fansi          0.5.0      2021-05-25 [1] CRAN (R 4.1.0)                  
 farver         2.1.0      2021-02-28 [1] CRAN (R 4.1.0)                  
 fastmap        1.1.0      2021-01-25 [1] CRAN (R 4.1.0)                  
 foreign        0.8-81     2020-12-22 [2] CRAN (R 4.1.0)                  
 Formula        1.2-4      2020-10-16 [1] CRAN (R 4.1.0)                  
 fs             1.5.0      2020-07-31 [1] CRAN (R 4.1.0)                  
 generics       0.1.0      2020-10-31 [1] CRAN (R 4.1.0)                  
 ggplot2      * 3.3.5      2021-06-25 [1] CRAN (R 4.1.0)                  
 glue           1.4.2      2020-08-27 [1] CRAN (R 4.1.0)                  
 gridExtra      2.3        2017-09-09 [1] CRAN (R 4.1.0)                  
 gtable         0.3.0      2019-03-25 [1] CRAN (R 4.1.0)                  
 Hmisc          4.5-0      2021-02-28 [1] CRAN (R 4.1.0)                  
 hms            1.1.0      2021-05-17 [1] CRAN (R 4.1.0)                  
 htmlTable      2.2.1      2021-05-18 [1] CRAN (R 4.1.0)                  
 htmltools      0.5.1.1    2021-01-22 [1] CRAN (R 4.1.0)                  
 htmlwidgets    1.5.3      2020-12-10 [1] CRAN (R 4.1.0)                  
 httpuv         1.6.1      2021-05-07 [1] CRAN (R 4.1.0)                  
 httr           1.4.2      2020-07-20 [1] CRAN (R 4.1.0)                  
 igraph         1.2.6      2020-10-06 [1] CRAN (R 4.1.0)                  
 jpeg           0.1-9      2021-07-24 [1] CRAN (R 4.1.0)                  
 jsonlite       1.7.2      2020-12-09 [1] CRAN (R 4.1.0)                  
 knitr          1.33       2021-04-24 [1] CRAN (R 4.1.0)                  
 labeling       0.4.2      2020-10-20 [1] CRAN (R 4.1.0)                  
 later          1.2.0      2021-04-23 [1] CRAN (R 4.1.0)                  
 lattice        0.20-44    2021-05-02 [2] CRAN (R 4.1.0)                  
 latticeExtra   0.6-29     2019-12-19 [1] CRAN (R 4.1.0)                  
 lifecycle      1.0.0      2021-02-15 [1] CRAN (R 4.1.0)                  
 lubridate      1.7.10     2021-02-26 [1] CRAN (R 4.1.0)                  
 magrittr     * 2.0.1      2020-11-17 [1] CRAN (R 4.1.0)                  
 Matrix         1.3-3      2021-05-04 [2] CRAN (R 4.1.0)                  
 memoise        2.0.0      2021-01-26 [1] CRAN (R 4.1.0)                  
 munsell        0.5.0      2018-06-12 [1] CRAN (R 4.1.0)                  
 nnet           7.3-16     2021-05-03 [2] CRAN (R 4.1.0)                  
 openssl        1.4.4      2021-04-30 [1] CRAN (R 4.1.0)                  
 pillar         1.6.2      2021-07-29 [1] CRAN (R 4.1.0)                  
 pkgbuild       1.2.0      2020-12-15 [1] CRAN (R 4.1.0)                  
 pkgconfig      2.0.3      2019-09-22 [1] CRAN (R 4.1.0)                  
 pkgload        1.2.1      2021-04-06 [1] CRAN (R 4.1.0)                  
 plyr           1.8.6      2020-03-03 [1] CRAN (R 4.1.0)                  
 png            0.1-7      2013-12-03 [1] CRAN (R 4.1.0)                  
 prettyunits    1.1.1      2020-01-24 [1] CRAN (R 4.1.0)                  
 processx       3.5.2      2021-04-30 [1] CRAN (R 4.1.0)                  
 progress       1.2.2      2019-05-16 [1] CRAN (R 4.1.0)                  
 promises       1.2.0.1    2021-02-11 [1] CRAN (R 4.1.0)                  
 ps             1.6.0      2021-02-28 [1] CRAN (R 4.1.0)                  
 purrr          0.3.4      2020-04-17 [1] CRAN (R 4.1.0)                  
 R6             2.5.0      2020-10-28 [1] CRAN (R 4.1.0)                  
 rappdirs       0.3.3      2021-01-31 [1] CRAN (R 4.1.0)                  
 RColorBrewer   1.1-2      2014-12-07 [1] CRAN (R 4.1.0)                  
 Rcpp           1.0.7      2021-07-07 [1] CRAN (R 4.1.0)                  
 RCurl          1.98-1.3   2021-03-16 [1] CRAN (R 4.1.0)                  
 remotes        2.4.0      2021-06-02 [1] CRAN (R 4.1.0)                  
 rlang          0.4.11     2021-04-30 [1] CRAN (R 4.1.0)                  
 rpart          4.1-15     2019-04-12 [2] CRAN (R 4.1.0)                  
 rprojroot      2.0.2      2020-11-15 [1] CRAN (R 4.1.0)                  
 rstudioapi     0.13       2020-11-12 [1] CRAN (R 4.1.0)                  
 rtweet       * 0.7.0.9000 2021-08-12 [1] Github (mkearney/rtweet@69ae100)
 scales       * 1.1.1      2020-05-11 [1] CRAN (R 4.1.0)                  
 sessioninfo    1.1.1      2018-11-05 [1] CRAN (R 4.1.0)                  
 stringi        1.6.2      2021-05-17 [1] CRAN (R 4.1.0)                  
 stringr      * 1.4.0      2019-02-10 [1] CRAN (R 4.1.0)                  
 survival       3.2-11     2021-04-26 [2] CRAN (R 4.1.0)                  
 testthat       3.0.4      2021-07-01 [1] CRAN (R 4.1.0)                  
 textutils      0.2-1      2021-04-01 [1] CRAN (R 4.1.0)                  
 tibble         3.1.3      2021-07-23 [1] CRAN (R 4.1.0)                  
 tidyselect     1.1.1      2021-04-30 [1] CRAN (R 4.1.0)                  
 usethis      * 2.0.1      2021-02-10 [1] CRAN (R 4.1.0)                  
 utf8           1.2.2      2021-07-24 [1] CRAN (R 4.1.0)                  
 vctrs          0.3.8      2021-04-29 [1] CRAN (R 4.1.0)                  
 vosonSML     * 0.29.13    2020-07-18 [1] CRAN (R 4.1.0)                  
 withr          2.4.2      2021-04-18 [1] CRAN (R 4.1.0)                  
 xfun           0.24       2021-06-15 [1] CRAN (R 4.1.0)

**Comment by llrs** [comment]
Not sure what is going on, currently worked fine for me. Maybe it is related to  problems detected with tweets with coordinates (See the work in progress on #627). 
If you find that error again, could you run `traceback()` and report its output back here? It would help to detect where does the error raise. Thanks!

**Comment by nicoleprause** [comment]
I feel it is very likely related to the geo NULL's from that other thread. The user pulls definitely faulted because some users did not have those data (I had to loop to check each user and add empty columns for missing). Ultimately, I got the script working with a combo of the older Rtweet and the newer RTweet. If I see it again, I will definitely run the trace.

Thanks for working on this. Pulling millions of records I still benefitted from the new script. Maybe some day parallelized. :)
